### PR TITLE
Add v1.0 documentation

### PR DESCRIPTION
**Pull request by ohmtech-rdi**
This PR integrates version 1.0 of the documentation.

### CODE REVIEW

File: reference/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+| Vshpe	 | • | • | Will tend to produce a higher pitched sound. |
+| Square	 |   | • | Waveshaped rectifier. Will tend to dry the signal much more than Rectifier. |
+| Lowfi	 | • |   | Slew rate limiter. This is an extreme version of a physical limitation that exists on any amplifier. {guilabel}`MOD` will change the slew rate limit, raising it will give more highs. |
+| Lowrfi	 | • | • | Waveshaped slew rate distortion. |
+| BrwnNz	 | • |   | Random walk on a waveshaper. Mod at 0 will create noise. |
+| Filter	 | • | • | Waveshaper followed by a resonator low pass filter. The higher the distortion gain, the higher the resonance. {guilabel}`MOD` will change the cutoff frequency. |
+| DwnSpl	 | • | • | Emulates an analogue sample & hold. {guilabel}`MOD` will change the hold time. | 
+| Bitcrsh | • | • | Crushes the bit resolution of the signal. May oscillate when used in the Xxx mode. |
+| Jelly	 | • | • | Enveloppe Follower driven Distortion. |
+| Fractl	 | • | • | Uses a chaotic feedback equation to to distort the sound. A bit experimental, pretty weird and analogue sounding. {guilabel}`MOD` will change the recursivity depth. Use some {guilabel}`BIAS` to make it unstable. |
```
Comment (daFunkyCoder) [inline reference/index.md:30]: ```to to```

**Review by daFunkyCoder — COMMENTED** [review]
Nits

File: performer/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Latch, the default option, which allows the knob to override the current sequencer step value as soon as it is turned, until another preset is loaded,
+- Relative, which allows to modulate slightly around the sequencer step value using the knob,
+- Override, which ignores the sequencer step knob value.
+
+To select the current sequence, press the {guilabel}`TYPE` encoder for 2 seconds, navigate to {guilabel}`PERFORMER`, then {guilabel}`CUR SEQ`. Press the encoder. By rotating the {guilabel}`TYPE` encoder choose the current sequence from 1 to 16. Once selected, navigate to {guilabel}`BACK`, press the encoder, and repeat this to go back to the initial screen.
+
+To save the current configuration of the knobs and switches, first make sure you are in {guilabel}`PRODUCER` mode. Then press the {guilabel}`TYPE` encoder two times quickly, and the "save step" menu will appear. The first line shows the current selected sequence and allows to exit without saving the step. To save a step into the sequence, rotate the encoder to the desired step and press the encoder. The step is then saved and the module will go back to the initial screen.
```
[the remark this answers is not in the document]
Comment (ohmtech-rdi) [reply performer/index.md:36]: What do you mean?

[the remark this answers is not in the document]
Comment (daFunkyCoder) [reply performer/index.md:36]: The back is not on the first line anymore.To exit without saving you need to scroll to BACK.

[the remark this answers is not in the document]
Comment (ohmtech-rdi) [reply performer/index.md:36]: Ah, but that's for version 1.1 only.

**Pull request merged**